### ISSUE TITLE
Vector elements floating

### ISSUE THREAD

**Issue by louwers** [maplibre/maplibre-native#3085]
### MapLibre Android Version

1.6.1-vulkan-pre1

### Android SDK Version

?

### Device

?

### What happened?

Reported by @mtpdog

> The only small issue I've noticed so far with 11.6.1-vulkan-pre1 is that all vector elements (vector maps, annotations) do not "stick" firmly to the raster background on quick camera movements. They kind of floating around their normal positions. Probably the vector and raster parts of rendering are not synchronized perfectly. Would be great if this is fixed as well.

### Steps to reproduce

Unclear but @adrian-cojocaru seems to be on it.

@mtpdog if you could, please clarify

### Renderer

Vulkan

### Relevant log output

```shell

```

**Comment by mtpdog** [comment]
This issue seems to be fixed in 11.7.0.

**Comment by louwers** [comment]
@mtpdog Also with Vulkan?

**Comment by mtpdog** [comment]
> [@mtpdog](https://github.com/mtpdog) Also with Vulkan?

Yes. I've tested it with Vulkan and OpenGL. Both work fine! (there was no issue with OpenGL anyway).
